### DISCUSSION TITLE
Switching to Pc, don't wanna say goodbye!

### DISCUSSION THREAD

**Discussion by Patchwxrk** [keiwando/toonsquid#44] [Q&A] [answered]
I adore toonsquid, it's simple enough for me to understand while having all the features i crave and with the upcoming updates im very excited, however this year for christmas im getting a drawing tablet. Is there any possible way you can toonsquid multi-platform sometime soon? love the app.

**Comment by keiwando** [comment, marked as answer]
I currently don't have any plans for adding ToonSquid to more platforms, sorry.
